### PR TITLE
Enable/disable addon signing for debug purposes

### PR DESCRIPTION
**Pull request by spacifici**

### CODE REVIEW

File: mozilla-release/mobile/android/confvars.sh [hunk 1]
```diff
@@ -41,4 +41,4 @@ MOZ_USE_NATIVE_POPUP_WINDOWS=1
 MOZ_APP_ID={aa3c5121-dab2-40e2-81ca-7ea25febc110}
 
 # Enable checking that add-ons are signed by the trusted root
-MOZ_ADDON_SIGNING=1
+MOZ_ADDON_SIGNING=`[ -n "${CLIQZ_DEBUG}" ] && echo 0 || echo 1`
```
Comment (chrmod) [inline mozilla-release/mobile/android/confvars.sh]: how about `MOZ_ADDON_SIGNING="${CLIQZ_DEBUG=0}"`  ?

Comment (spacifici) [reply mozilla-release/mobile/android/confvars.sh]: Well, it's a different semantic, what I want to have is:
CLIQZ_DEBUG=<anyvalue> -> MOZ_ADDON_SIGNING=0
CLIQZ_DEBUG unset or empty -> MOZ_ADDON_SIGNING=1

Comment (spacifici) [reply mozilla-release/mobile/android/confvars.sh]: @chrmod probably this is better:
```bash
MOZ_ADDON_SIGNING=`[ "$CLIQZ_DEBUG" -ne 1 ] && echo 1 || echo 0`
```

**Comment by alver-cliqz** [comment]
looks ok for me, as I told before - it's ok to do stuff in any way, later you can fix if it will not fit into your needs.

File: mozilla-release/mobile/android/confvars.sh (hunk 1 above)
Comment (chrmod) [reply mozilla-release/mobile/android/confvars.sh]: :+1:

**Pull request merged**